gql/transport/exceptions.py

```diff
@@ -22,9 +25,15 @@ class TransportQueryError(Exception):
     This exception should not close the transport connection.
     """
 
-    def __init__(self, msg, query_id=None):
+    def __init__(
+        self,
+        msg: str,
+        query_id: Optional[int] = None,
+        errors: Optional[List[Any]] = None,
```

Comment:
Everything is okay, but I think we could provide a better type as `TransportQueryError` is only used at `client.py` where it receives `Optional[List[GraphQLError]]` and `websockets.py` which receives `Any` as type (due to json parsing).

Not sure if possible.

Reply:
The problem is that currently there is no validation done of the errors returned from the different possible types of backends. We could try to convert the errors received to `GraphQLError` but it could break existing code if the error received is not compatible.

Reply:
Gotcha! Let's then merge this as it is.